libkbfs/test_common.go

```diff
@@ -79,9 +79,10 @@ func MakeTestBlockServerOrBust(t logger.TestLogBackend,
 	}
 }
 
-// MakeTestConfigOrBust creates and returns a config suitable for
-// unit-testing with the given list of users.
-func MakeTestConfigOrBust(t logger.TestLogBackend,
+// MakeTestConfigOrBustLoggedIn creates and returns a config suitable for
+// unit-testing with the given list of users. loggedInIndex specifies the
+// index (in the list) of the user being logged in
+func MakeTestConfigOrBustLoggedIn(t logger.TestLogBackend, loggedInIndex int,
```

Comment:
This is to used to clean up the tests in kbfs-server/bserver. The cleaning of tests is orthogonal to putAgain, but the old tests no longer compiled after I vendored in new kbfs changes and I thought it's a good time to clean up those tests...